### PR TITLE
Added a way to check if a numeric type cell is NaN

### PR DESCRIPTION
**Pull request by antonini**
- [ ] closes #xxxx
- [ ] tests added / passed
- [ ] passes `black pandas`
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

File: pandas/io/excel/_xlrd.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
 
 from pandas.io.excel._base import _BaseExcelReader
 
+from pandas.core.dtypes.missing import isnull
```
Comment (jbrockmendel) [inline pandas/io/excel/_xlrd.py]: use `isna`

Comment (jbrockmendel) [reply pandas/io/excel/_xlrd.py]: also this import would go above the previous one

**Comment by jbrockmendel** [comment]
is there an issue describing why the current behavior is wrong?  can you add a test?

**Comment by WillAyd** [comment]
I am -1 on converting NA values to 0 - is there an open issue about this?

**Comment by WillAyd** [comment]
@antonini thanks for the interest in submitting a PR! I don't know if this is a change we actually want to make though. If you feel strongly about it I'd suggest opening an issue first and can be discussed there

**Pull request closed without merging**